### PR TITLE
Fix/development/remove confirmation

### PR DESCRIPTION
**Pull request by MarcF12**
Default delete confirmation

### CODE REVIEW

File: Hestia/Hestia/AppDelegate.cs [hunk 1]
```diff
@@ -17,11 +25,115 @@ public override UIWindow Window
...
             // Override point for customization after application launch.
             // If not required for your application you can safely delete this method
+            NSUserDefaults userDefaults = NSUserDefaults.StandardUserDefaults;
+
+            // For Debugging
+           // Globals.ResetUserDefaults();
+           // userDefaults.RemoveObject(strings.defaultsServerNameHestia);
+           // userDefaults.RemoveObject(strings.defaultsIpHestia);
+           // userDefaults.RemoveObject(strings.defaultsPortHestia);
+           // userDefaults.RemoveObject(strings.defaultsLocalHestia);
```
Comment (Mikmist) [inline Hestia/Hestia/AppDelegate.cs]: Please do not have commented out code.

File: Hestia/Hestia/UIViewControllerAuth0.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+        public override void ViewDidLoad()
+        {
+            base.ViewDidLoad();
+            // Task<LoginResult> loginResult = GetLoginResult();
+
+            //userDefaults.SetString(loginResult.IdentityToken, strings.defaultsIdentityTokenHestia);
+            //userDefaults.SetString(loginResult.AccessToken), strings.defaultsAccessTokenHestia);
+        
+            // PerformSegue...
```
Comment (Mikmist) [inline Hestia/Hestia/UIViewControllerAuth0.cs]: Same here

File: Hestia/Hestia/AppDelegate.cs (hunk 1 above)
Comment (MarcF12) [reply Hestia/Hestia/AppDelegate.cs]: I will still need this when making the Segues after you add the auth0 code.

File: Hestia/Hestia/UIViewControllerAuth0.cs (hunk 2 above)
Comment (MarcF12) [reply Hestia/Hestia/UIViewControllerAuth0.cs]: Same here.

**Pull request merged**